lopper/assists/subsystem.py

```diff
@@ -860,8 +860,10 @@ def cpu_expand( tree, subnode, verbose = 0):
         for n in subnode.subnodes():
             if n.name == "domain-to-domain":
                 n + LopperProp(name="cluster_cpu", value=cluster_cpu)
-                n.resolve()
-                n.sync()
+
+    pd_prop_node = [ n for n in cluster_node.subnodes() if n.propval("power-domains") != [''] ]
+    if len(pd_prop_node) == 1:
+        subnode + LopperProp(name="rpu_pd_val", value=pd_prop_node[0].propval("power-domains"))
```

Comment:
The problem with this, is that subsystem isn't only used by openamp. It is a generalized application of the system device tree subsystem specification.

So something like this really belongs in an openamp specific assist, or somehow be data driven on more than the name of the node. This is all very specific to a node layout.

Reply:
hi @zeddii  the reason i added these parts (RPU only btw) is for AMD-xilinx specific case where some of the rpu information is used AFTER RPUs are removed by domain access in the openamp case.

Reply:
Hi @zeddii  i get your point about what is being stored.

instead of adding another lop or assist (which makes it harder to debug and track) can we move this information processing to singular routine, like openamp_remote_domain_expand(), that stores the information in the openamp remoteproc relation? 

that way the logic is in 1 place but we are acting to reduce the spaghetti code effect :) 

please advise!